src/components/PokemonDetails/PokemonDetails.js

```diff
@@ -21,6 +21,21 @@ const PokemonDetails = ({ foundPokemon, getPokemonImage }) => {
       setError(error.message);
     }
   };
+
+  const filterMoves = () => {
+    const result = pokemonDetails.moves.map((move) => {
+      const version = move.version_group_details[0].version_group.name
+      const name = move.move.name.split("-").join(" ");
+
+      if (version === 'red-blue') {
+        return name;
+      }	
+    }).filter(move => {
+      return move !== undefined
+    }).sort((elementA, elementB) => elementA - elementB)
+    return result;
+  }  
+
```

Comment:
Thank you for being attentive to those details

Reply:
Yes! nice work!